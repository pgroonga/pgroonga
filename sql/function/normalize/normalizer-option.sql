SELECT pgroonga_normalize('aBcDe 123', 'NormalizerAuto');
