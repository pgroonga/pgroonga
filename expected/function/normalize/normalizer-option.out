SELECT pgroonga_normalize('aBcDe 123', 'NormalizerAuto');
 pgroonga_normalize 
--------------------
 abcde 123
(1 row)

